lib/panoptes/client/collections.rb

```diff
@@ -3,6 +3,12 @@
 module Panoptes
   class Client
     module Collections
+      def collection(collection_id)
+        response = panoptes.get("/collections/#{collection_id}")
+        response.fetch('collections')
+                .find { |i| i.fetch('id').to_s == collection_id.to_s }
```

Comment:
Not blocking: The API will only return 1 resource if we get with a known resource id (uses API controller show route), if it can't find it i'll return and empty collections respone. I'm a little surprised this fetch by id matching is in here tbh, though i see it's been copied from another spot in code. It doesn't hurt but it doesn't need to be there imho.

Reply:
You are right - it also makes that line of code more confusing. Ive changed this bit of code to be more in line with what that endpoint is actually returning, thanks for bringing it up 👍